src/main/java/com/gengo/client/GengoConstants.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * 
+ */
+package com.gengo.client;
+
+/**
+ * @author hiroyu-n@inter-plug.co.jp
+ *
+ */
+public final class GengoConstants {
+	protected static String BASE_URL_STANDARD = "http://api.gengo.com/v2/";
```

Comment:
This one came from your own original code. https://github.com/gengo/gengo-java/blob/master/src/main/java/com/gengo/client/GengoClient.java#L26

Anyway, I'll update it. Thanks.